### PR TITLE
Add ShareApplication explanation on README.md

### PR DESCRIPTION
**Pull request by ygorbarboza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,24 @@ Share Social , Sending Simple Data to Other Apps
     ```
       compile project(':react-native-share')
     ```
+5. Follow this [guide](https://developer.android.com/training/secure-file-sharing/setup-sharing.html)
+6. Make your `Application` class implements `ShareApplication`
+ - Make `getFileProviderAuthority` function return the `android:authorities` that was added on AndroidManifest file
+
+Example: 
+```
+class MyApplication extends Application implements ShareApplication, ReactApplication {
```
Comment (bitshadow) [inline README.md]: How do we import ShareApplication?

File: README.md [hunk 2]
```diff
@@ -49,6 +49,20 @@ Share Social , Sending Simple Data to Other Apps
...
 
+Example: 
+```
+class MyApplication extends Application implements ShareApplication, ReactApplication {
+
+...
+
+     @override
+     String getFileProviderAuthority() {
+            return <File provider authority>
```
Comment (sibelius) [inline README.md]: could you provide an example of this?

File: README.md (hunk 1 above)
Comment (ygorbarboza) [reply README.md]: I added the import on the example

File: README.md (hunk 2 above)
Comment (ygorbarboza) [reply README.md]: this string should be the same you added in the AndroidManifest. If a add a fixed string on the example can be misleading

Comment (sibelius) [reply README.md]: got it

**Comment by bitshadow** [comment]
Thanks @ygorbarboza its working for me now.

**Pull request merged**